db/data/20211129150253_backfill_funding_methods_for_previous_academic_cycle.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class BackfillFundingMethodsForPreviousAcademicCycle < ActiveRecord::Migration[6.1]
+  def up
+    FundingMethod.where(academic_cycle_id: nil).each do |funding_method|
```

Comment:
Since all the funding methods are for the last year's cycle, we can just do the following:

```suggestion
    FundingMethod.update_all(academic_cycle_id: 6)
```